<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCtrlMainWindow</class>
 <widget class="QMainWindow" name="CCtrlMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCtrlMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_ProgLoad">
       <property name="text">
        <string>转圈加载中</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_DashBoard">
       <property name="text">
        <string>仪表盘1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_DashBoard1">
       <property name="text">
        <string>仪表盘2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ProgWater">
       <property name="text">
        <string>水波纹进度</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="CCtrlDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>95</width>
      <height>95</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CCtrlDateEdit</class>
   <extends>QDateEdit</extends>
   <header>CCtrlDateEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
